week3/homework/src/index.js

```diff
@@ -3,80 +3,81 @@
 const express = require('express');
 const Joi = require('joi');
 const app = express();
-const todos = require('./todos');
+const validateTodos = require('./validateTodos');
+const todo = require('./todo.json');
```

Comment:
Please dont read json files using require. It is a really bad practice. Use `fs.readFile` like we did in the lectures.

Reply:
Thanks . I will modify it